tests/python/mock_jira.py

```diff
@@ -473,6 +474,16 @@ def projects(self):
         """
         return self._projects
 
+    def project(self, project_id):
+        """
+        Mocked Jira method
+        Return a :class:`JiraProject`
+        """
+        for project in self._projects:
+            if project.key == project_id:
+                return project
```

Comment:
I feel like being fancy...but you can ignore me if you prefer. ;)

`project = next((p for p in self._projects if p.key == project_id), None)`

Reply:
I like readability. 😄